sdk/test/openai_api/validate_format.py

```diff
@@ -0,0 +1,26 @@
+import json
+import os
+
+from format_finder import extract_reference, fetch_api_format
+
+print(
+    "====================== Start validating the OpenAI API reference format"
+    " ======================"
+)
+result = fetch_api_format()
+
+reference_path = os.path.abspath("./test/openai_api/reference_format.json")
+with open(reference_path, "r") as f:
+    reference = json.load(f)
+    if result.find(reference) != -1:
+        print("The OpenAI API format is up-to-date.")
+    else:
+        message = (
+            "The OpenAI API format has changed. Run "
```

Comment:
Should devs also be told to check if the changes will affect any Omnilog components, before updating the reference?

Reply:
Good call